django/contrib/messages/storage/cookie.py

```diff
@@ -57,6 +58,11 @@ class CookieStorage(BaseStorage):
     max_cookie_size = 2048
     not_finished = '__messagesnotfinished__'
 
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        key_salt = 'django.contrib.messages'
```

Comment:
If you move this into the class scope it will not be duplicated below and might allow for easier customization